MarvelApiClient/Parser.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Parser.swift
+//  MarvelApiClient
+//
+//  Created by Pedro Vicente Gomez on 16/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+protocol Parser {
```

Comment:
Sure, but we don't need it now and the implementation will be empty. I can add it if you want :)